dispander/module.py

```diff
@@ -18,21 +29,87 @@ async def on_message(self, message):
             return
         await dispand(message)
 
+    @commands.Cog.listener()
+    async def on_raw_reaction_add(self, payload: discord.RawReactionActionEvent):
+        await delete_dispand(self.bot, payload=payload)
+
+
+async def delete_dispand(bot: discord.Client,
+                         *,
+                         payload: Optional[discord.RawReactionActionEvent] = None,
+                         reaction: Optional[discord.Reaction] = None,
+                         user: Optional[discord.User] = None):
```

Comment:
```suggestion
                         user: Optional[discord.User] = None) -> None:
```
引数に型ヒントがあって、返り値に型ヒントが無いのは何故？

Reply:
わかりにくかったところにつけてあるだけです
ライブラリの方針としてもつけてなさそうなのでいらないかなと